gensim/models/doc2vec.py

```diff
@@ -375,60 +318,76 @@ def dbow(self):
         """
         return self.sg  # same as SG
 
-    def _set_train_params(self, **kwargs):
-        pass
+    @property
+    @deprecated("The `docvecs` property has been renamed `dv`.")
+    def docvecs(self):
```

Comment:
Do we want to bother with these warnings? We'll have to go back and remove it later. I think because this is a major release, we can afford to break stuff.

I think it would be simpler to document this in the release notes.